generator/src/generator/namespace.rs

```diff
@@ -156,8 +156,9 @@ pub(super) fn generate_request_reply_enum(
         outln!(out, "}}");
         outln!(
             out,
-            "// Get the matching reply parser (if any) for this request"
+            "/// Get the matching reply parser (if any) for this request"
```

Comment:
Just one last detail: add a stop at the end of the sentence, otherwise it would show as "Get the matching reply parser (if any) for this request For `Request::Unknown`, `None` is also returned." in the generate documentation.

Reply:
Ok.